fingertip/plugins/os/centos-stream.py

```diff
@@ -0,0 +1,104 @@
+import os
+
+import fingertip.machine
+from fingertip.util import path
+
+#[Repos]
+#http://ftp.fi.muni.cz/pub/linux/centos/8-stream/
+#official mirror: http://mirror.centos.org/centos/8-stream/
+
+REPO = 'http://mirror.centos.org/centos/'
+
+def main(m=None, fips=False):
+    m = m or fingertip.build('backend.qemu')
+    if hasattr(m, 'qemu'):
+        return m.apply(install_in_qemu, fips=fips)
+    raise NotImplementedError()
+
+
+def install_in_qemu(m=None,
+                    fips=False):
```

Comment:
Please remove the `fips` parameter, I don't see anyone using it in the close future.
Or allow passing custom stuff to `-append` instead.